build/macos/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh
+set -e
+
+version=$1
+
+mkdir Volume
+osacompile -o Volume/Spicetify.app main.applescript
+rm -rf Volume/Spicetify.app/Contents/_CodeSignature
```

Comment:
Again, like I have said in discord, do not remove it regardless

Reply:
There is no reason to keep it in, if anything it will cause more errors when we start mutating more generated files inside the .app